src/MetaMorpho.sol

```diff
@@ -737,6 +744,8 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
 
                 marketConfig.enabled = true;
 
+                _updateLastTotalAssets(lastTotalAssets + MORPHO.expectedSupplyAssets(marketParams, address(this)));
```

Comment:
Consider avoiding executing this function if `MORPHO.expectedSupplyAssets(marketParams, address(this))` is equal to zero. 

I would say that the normal scenario should be that you are enabling a market where no one has supplied on behalf of MM.

Reply:
True, but it would save only ~2k gas since we still need to call `expectedSupplyAssets`. In the current configuration we're adding one cold `SLOAD` and one warm `SSTORE` that is not changing the value. So no sure it's worth considering this case.

Reply:
I'm not sure it's worth it considering this case. The current implementation only adds one cold `SLOAD` and one warm `SSTORE` so around 2k gas.